src/components.js

```diff
@@ -0,0 +1,16 @@
+import { ElementMixin } from './ElementMixin';
+import { ContainerMixin } from './ContainerMixin';
+
+export const SortableList = {
+  mixins: [ ContainerMixin ],
+  render (h) {
+    return h('div', this.$slots.default);
+  },
+};
+
+export const SortableItem = {
+  mixins: [ ElementMixin ],
+  render (h) {
+    return h('div', this.$slots.default);
+  },
+};
```

Comment:
Hmm, looks like this file needs a newline. If this isn't enforced by eslint, it should be! I'll look into this unless you want to add the rule and see where else the code is missing newlines :)